### ISSUE TITLE
Loss Visualization using Tensorboard and Resume Training from last checkpoint

### ISSUE THREAD

**Issue by rm2886** [WenmuZhou/DBNet.pytorch#26]
@WenmuZhou can we visualize the losses on tensorboard?
please explain if it's possible.
And how can we resume training from the last checkpoint?

**Comment by WenmuZhou** [comment]
tensorboard：https://github.com/WenmuZhou/DBNet.pytorch/blob/master/config/icdar2015_dcn_resnet18_FPN_DBhead_polyLR.yaml#L52


resume：https://github.com/WenmuZhou/DBNet.pytorch/blob/master/config/icdar2015_dcn_resnet18_FPN_DBhead_polyLR.yaml#L49

**Comment by rm2886** [comment]
Thanks @WenmuZhou
